src/core/plugins/layout/spec-extensions/wrap-selector.js

```diff
@@ -9,7 +9,7 @@ export const taggedOperations = (oriSelector, system) => (state, ...args) => {
   // Filter, if requested
   let filter = layoutSelectors.currentFilter()
   if (filter) {
-    if (filter !== true && filter !== "true" && filter !== "false") {
+    if (filter !== true) {
```

Comment:
Without this change, when typing "true" or "false" into the filter input, we'd not filter anything at all, as in we'd still have all the operations. 

Now it should actually search for operations with "true" or "false". Here where there are no such operations:

<img width="1498" alt="Screenshot 2024-04-16 at 14 15 16" src="https://github.com/swagger-api/swagger-ui/assets/44094416/c40c3061-0d8f-4396-9e6a-d4eb33ac1b32">

Reply:
Nice touch!